sc/social/like/profiles/default/registry.xml

```diff
@@ -1,4 +1,15 @@
 <?xml version="1.0"?>
 <registry>
   <records interface="sc.social.like.controlpanel.ISocialLikeSettings" />
+
+  <record name="plone.app.tiles">
```

Comment:
I think not. If sc.social.like is installed and then the cover is installed, this record needs to be there.